llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -44,14 +44,19 @@
 #include "llvm/Transforms/Utils/Cloning.h"
 
 #include <algorithm>
+#include <map>
 #include <memory>
 #include <string>
 #include <vector>
 
 using namespace llvm;
 
 using string_vector = std::vector<std::string>;
+using FuncPtrVector = std::vector<const Function *>;
+using EntryPointsSet = std::map<StringRef, FuncPtrVector>;
```

Comment:
```suggestion
using EntryPointsMap = std::map<StringRef, FuncPtrVector>;
```
?
Type and variable name on line 322 (and 928) are not consistent.

Reply:
Thank you, done.